roshambo-ai/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.7.3-slim
+COPY requirements.txt /
+RUN pip3 install --upgrade pip
+RUN pip3 install -r /requirements.txt
+
+COPY . /app
```

Comment:
I'm probably just not seeing something, but where's the code for this?

Reply:
ooops, that was an accidental copy of the existing one in the kubernetes folder.  will delete.  I thought it was kind of weird to live in the kubernetes folder when I moved the yamls to the kustomize directory